### PR TITLE
Cubing recipes

### PR DESCRIPTION
**Pull request by nullptr-ref**
Added cubing_recipes functionality - Cube flawlesses to perfects, cube essenses into token
Added cubing recipes as configurable option in settings.
Updated code to switch to correct stash page when trying to interact with the cube

### CODE REVIEW

File: internal/action/stash.go [hunk 1]
```diff
@@ -212,3 +212,16 @@ func (b *Builder) switchTab(tab int) {
 	b.HID.Click(game.LeftButton, x, y)
 	helper.Sleep(500)
 }
+
+func (b *Builder) SwitchTab(tab int) *Chain {
```
Comment (hectorgimenez) [inline internal/action/stash.go:216]:
```suggestion
func (b *Builder) SwitchStashTab(tab int) *Chain {
```

Builder struct is shared across multiple parts of the application, better to be specific.

File: internal/server/http_server.go [hunk 2]
```diff
@@ -312,6 +312,11 @@ func (s *HttpServer) characterSettings(w http.ResponseWriter, r *http.Request) {
 		// Gambling
 		cfg.Gambling.Enabled = r.Form.Has("gamblingEnabled")
 
+		// Cube Recipes
+		cfg.EnableCubeRecipes = r.Form.Has("enableCubeRecipes")
+
+		// Companion
```
Comment (hectorgimenez) [inline internal/server/http_server.go:318]: We don't need this?

Comment (nullptr-ref) [reply internal/server/http_server.go:318]: Some people might not want to enable the cube recipes to keep their flawless gems for other purposes. Additionally, we might add other recipes in the future that people don't want to waste their resources on.

Comment (divineblade7) [reply internal/server/http_server.go:318]: That was my thought as well, such as crafting recipes or cubing runes eventually. Having option to disable would be good.

**Pull request merged**